location/android/src/main/java/com/lyokone/location/FlutterLocation.java

```diff
@@ -99,6 +96,11 @@
         this.activity = activity;
     }
 
+    FlutterLocation(PluginRegistry.Registrar registrar) {
+        this(registrar.context(), registrar.activity());
+        registrar.addRequestPermissionsResultListener(requestPermissionsResultListener);
```

Comment:
I think we can just do 
```suggestion
        registrar.addRequestPermissionsResultListener(this);
```
And remove the `requestPermissionsResultListener` below since this class is already extending PluginRegistry.RequestPermissionsResultListener :)